datavec/datavec-python/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2015-2018 Skymind, Inc.
+  ~
+  ~ This program and the accompanying materials are made available under the
+  ~ terms of the Apache License, Version 2.0 which is available at
+  ~ https://www.apache.org/licenses/LICENSE-2.0.
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations
+  ~ under the License.
+  ~
+  ~ SPDX-License-Identifier: Apache-2.0
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>datavec-parent</artifactId>
+        <groupId>org.datavec</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <packaging>jar</packaging>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>datavec-python</artifactId>
+
+
+    <dependencies>
+
+     <dependency>
```

Comment:
Mind adding dependency versions under dependency management in the top level pom? We do this to avoid dependency conflicts across modules.
Do this with 
<dependency>
            <groupId>org.bytedeco.javacpp-presets</groupId>
            <artifactId>cpython-platform</artifactId>
            <version>3.6-1.4.4-SNAPSHOT</version>
        </dependency>
        <dependency> 


converting to:
<dependency>
            <groupId>org.bytedeco.javacpp-presets</groupId>
            <artifactId>cpython-platform</artifactId>
        </dependency>
        <dependency>\

putting the previous declaration under a dependency management section.

Also, please version the python and javacpp versions with variables at the top level.